ui/tutorial-renderer/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'com.jakewharton.butterknife'
+
+android {
+    dataBinding {
+        enabled = true
+    }
+}
+
+dependencies {
+    implementation project(':library:base')
+
+    implementation project(':ui:base-tool')
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${deps.kotlin}"
+
+    implementation "androidx.fragment:fragment-ktx:${deps.androidX.fragment}"
+    implementation "androidx.lifecycle:lifecycle-extensions:${deps.androidX.lifecycle}"
+    implementation "androidx.constraintlayout:constraintlayout:${deps.androidX.constraintLayout}"
+    implementation "androidx.recyclerview:recyclerview:${deps.androidX.recyclerView}"
+
+    implementation "org.ccci.gto.android:gto-support-core:${deps.gtoSupport}"
+    implementation "org.ccci.gto.android:gto-support-picasso:${deps.gtoSupport}"
+    implementation "org.ccci.gto.android:gto-support-recyclerview:${deps.gtoSupport}"
+    implementation "org.ccci.gto.android:gto-support-util:${deps.gtoSupport}"
+    implementation "me.relex:circleindicator:${deps.circleindicator}"
+
+    implementation "com.google.guava:guava:${deps.guava}"
```

Comment:
what are you using guava for in this module? is it something that can be replaced by kotlin?